schemas/v1.0/schema.yaml

```diff
@@ -1,5 +1,101 @@
-$id: https://spec.openapis.org/overlay/1.0/schema/WORK-IN-PROGRESS
+$id: https://spec.openapis.org/overlay/1.0/schema/2024-10-17
 $schema: https://json-schema.org/draft/2020-12/schema
-description: The description of Overlay v1.0.x documents
-
+description: The description of OpenAPI Initiative Overlay v1.0.0 documents
+  without schema validation, as defined by
+  https://spec.openapis.org/overlay/v1.0.0
 type: object
+$comment: https://spec.openapis.org/overlay/v1.0.0#overlay-object
```

Comment:
Would we want to update all these `$comment`s when we publish v1.0.1?

I'd rather remove them all.

Reply:
just following OAS, I can remove them.

Reply:
Thanks for catching this, opened
* https://github.com/OAI/OpenAPI-Specification/issues/4181